File: Java/str1.java
import java.util.Scanner;

public class str1 {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {  
            System.out.println("Enter a string: ");
            String str = sc.nextLine();  

            int count = 0;
            for (int i = 0; i < str.length(); i++) {
                count++;
            }

            System.out.println("The length of the string is: " + count);
        } 
    }
}

File: Java_Practice/CPP/str1.cpp
#include<iostream>
#include<string>
using namespace std;
int main()
{
    char str[100];
    int count = 0;
    cout<<"enter a str:"<<endl;
    cin>>str;
    while(str[count] !='\0'){
        count++;
    }
    cout<<"the string length is : "<<count<<endl;
    return 0;

}